src/less/main.less

```diff
@@ -209,3 +209,44 @@
 #notebook {
   .transition-all;
 }
+
+
+/* chalkboard plugin and RISE buttons */
+#notescanvas { position: absolute; top: 0; }
+#chalkboard { display: none; }
+
+/* button toolbar */
+#help_b,
+#exit_b,
+.chalkboard-button {
+  color: @some-grey;
+  position: fixed !important;
+  z-index: 30;
+  cursor: pointer;
+}
+
+#exit_b {
+  top: 0.8em;
+  left: 0.6em;
+}
+
+#help_b {
+  left: 0.6em ;
+  bottom: 0.8em;
+}
+
+/* use !important to override stuff that
+   would normally be configured through
+   chalkboard options */
```

Comment:
And why you can't configure them using the chalkboard options? Because you use a set of config options by default and there is not an easy way to access the chalkboard config mechanism?

Reply:
again it's been a long time so I'm not 100% positive here
IIRC since we load the chalkboard extension as-is, it comes with settings in its own css, but we want to override these, hence `!important`